.github/workflows/ci.yml

```
@@ -93,19 +90,21 @@ jobs:
       contains(github.event.head_commit.message, '[release]')
     steps:
       - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
       - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v8.16.0
       - uses: Alfresco/alfresco-build-tools/.github/actions/free-hosted-runner-disk-space@v8.16.0
       - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v8.16.0
       - uses: Alfresco/alfresco-build-tools/.github/actions/configure-git-author@v8.16.0
         with:
-          username: ${{ env.GIT_USERNAME }}
-          email: ${{ env.GIT_EMAIL }}
+          GIT_USERNAME: ${{ vars.BOT_GITHUB_USERNAME }}
+          GIT_EMAIL: ${{ vars.BOT_GITHUB_EMAIL }}
```

```suggestion
          username: ${{ vars.BOT_GITHUB_USERNAME }}
          email: ${{ vars.BOT_GITHUB_EMAIL }}
```

input names are still the same :D